src/components/detail/detail.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { AiOutlineArrowLeft } from 'react-icons/ai';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import './detail.css';
```

Comment:
- [ ] Kinldy make sure you have the module `detail.css` in your directory to import it. It's actually the reason why your app is crashing

![image](https://user-images.githubusercontent.com/97100091/225730114-2bbba1aa-422d-461a-8bd2-2c93e77afa1c.png)